packages/web-react/src/components/UNSTABLE_EmptyState/stories/UNSTABLE_EmptyState.stories.tsx

```diff
@@ -0,0 +1,58 @@
+import { Markdown } from '@storybook/blocks';
+import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+import { ButtonLink } from '../../Button';
+import { Heading } from '../../Heading';
+import { Link } from '../../Link';
+import { Text } from '../../Text';
+import { UNSTABLE_ActionLayout } from '../../UNSTABLE_ActionLayout';
```

Comment:
```suggestion
import { ButtonLink, Heading, Link, Text, UNSTABLE_ActionLayout } from '../..';
```

Reply:
I think importing the component from the component file itself is better than from the index file which is a kind of syntax sugar for developers.